### PR TITLE
Set timeout for setup_kubernetes to 10 minutes

### PR DESCRIPTION
**Pull request by bergmannf**
With timeout 0 it will only check once and fail otherwise.

## Why is this PR needed?

Attempt to fix `setup_kubernetes_version` - spuriously fails when running nightly tests, especially on VMWare.

With the 10 minute timeout the pipeline on `vmware` seems to work.

# Merge restrictions

(Please do not edit this)

We are in *v4-maintenance phase*, so we will restrict what can be merged to prevent unexpected surprises:

    What can be merged (merge criteria):
        2 approvals:
            1 developer: code is fine
            1 QA: QA is fine
        there is a PR for updating documentation (or a statement that this is not needed)

### CODE REVIEW

**Comment by pablochacin** [comment]
@bergmannf I'm changing the way we check for pods to become ready which seems to be more robust that just increasing the timeout here. It is not yet merged, but should be soon. Look at this PR  https://github.com/SUSE/skuba/pull/801

File: ci/infra/testrunner/tests/utils.py [hunk 1]
```diff
@@ -31,7 +37,11 @@ def _handle_timeout(signum, frame):
         signal.signal(signal.SIGALRM, _handle_timeout)
         signal.alarm(timeout)
         try:
-            return func(*args, **kwargs)
+            start = timer()
```
Comment (Itxaka) [inline ci/infra/testrunner/tests/utils.py]: should we create a contextmanager to track time if we are gonna be using this down the line?

Comment (bergmannf) [reply ci/infra/testrunner/tests/utils.py]: I added a `contextmanager` that just yields and allow executing inside the block.

Thanks for the suggestion!

File: ci/infra/testrunner/utils/timit.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import contextlib
```
Comment (chentex) [inline ci/infra/testrunner/utils/timit.py]: is there a typo in the file name?
non-blocking

File: ci/infra/testrunner/utils/timit.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+from timeit import default_timer as timer
+
+
+logger = logging.getLogger('testrunner')
+
+
+@contextlib.contextmanager
+def timed(description: str) -> None:
+    start = timer()
+    yield
```
Comment (Itxaka) [inline ci/infra/testrunner/utils/timit.py]: fancy! contexmanager in python 3 is much nicer than in py2!

File: ci/infra/testrunner/vars.yaml [hunk 4]
```diff
@@ -8,7 +8,7 @@ nodeuser: "sles"  # Default node username
 skuba:
   binpath: "" # path to skuba binary
   srcpath: "" # path to skuba source
-  verbosity: 5 # default value passed to skuba with -v
+  verbosity: 10 # default value passed to skuba with -v
```
Comment (Itxaka) [inline ci/infra/testrunner/vars.yaml]: I guess this commit needs to be dropped as its marked as test?

**Comment by jordimassaguerpla** [comment]
Hi!
New code has been merged in master regarding terraform 0.12. Please rebase your branch to take this changes in.
Happy hacking

**Comment by chentex** [comment]
This PR is no longer relevant with the changes that the CI has gotten, closing it.

**Pull request closed without merging**